tts/UI.def.lua

```diff
@@ -149,6 +162,20 @@ UI = {}
 ---@field scrollSensitivity nil | number @Default 1.0. A factor/multiplier to augment the scroll speed.
 ---@field itemHeight nil | number
 
+---@shape tts__UIDropdownElement : tts__UIInputElementBase
+---@field tag "Dropdown"
+---@field attributes nil | tts__UIDropdownElement_Attributes
+---@field children nil | tts__UIOptionElement[]
+
+---@shape tts__UIImageElement_Attributes : tts__UIElementBase_Attributes
+---@field image string
+---@field preserveAspect tts__UIElement_Boolean
```

Comment:
I suspect this ought to be optional i.e. `nil | tts__UIElement_Boolean`.

Reply:
Changed.  I just love that the API says the default varies. :)